docs/api/api_docs.yaml

```
@@ -2027,91 +2043,220 @@ paths:
     get:
       responses:
         '200':
-          description: OK
+          description: Retrieved replication configurations successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/All-replications'
+        '404':
+          $ref: '#/components/responses/Not-found'
       tags:
         - Admin
+      summary: Get all replication configurations
+      description: |-
+        This will retrieve all the defined replication configs as a key map.
+
+        The `assigned_node` fields will end with `(local)` or `(non-local)` depending on if the replication is local or not.
     post:
       responses:
         '200':
-          description: OK
+          $ref: '#/components/responses/Replicator-updated'
+        '201':
+          $ref: '#/components/responses/Replicator-created'
+        '400':
+          description: Bad Request. Likely cause is that replicator was not stopped before attempting update.
```

Let's not speculate - and leave the reason for the message contained in the response body (which is already defined) as a response type.

```suggestion
          $ref: '#/components/responses/request-problem'
```